content/docs/SUSHI/running/_index.md

```diff
@@ -15,20 +15,12 @@ This documentation assumes you have a SUSHI-compliant project structure and conf
 SUSHI is executed from the command line. The general form of the SUSHI execution command is as follows:
 
 ```shell
-{{< terminal >}} sushi {options}
+{{< terminal >}} sushi {command} {options}
 ```
 
-Use `sushi --version` and `sushi --help` to get basic information about the current SUSHI version.
+Supported commands are `build`, `init`, `update-dependencies`, and `help`.
 
-{{% small-pageinfo color="primary" %}}
-<span class="tag">SUSHI 3.0</span>For SUSHI 3.0.0 and later, the general form of the SUSHI execution command is:
-
-```shell
-$ sushi {command} {options}
-```
-
-Supported commands are `build`, `init`, `update-dependencies`, and `help` (which replaces `sushi --help`).
-{{% /small-pageinfo %}}
+Use `sushi --version` and `sushi help` to get basic information about the current SUSHI version.
```

Comment:
The `help` command is already noted above and is pretty self-explanatory. It also doesn't say the version. So I say we drop that bit in this sentence since it's kind of awkward anyway.
```suggestion
Use `sushi --version` to get basic information about the current SUSHI version.
```

Reply:
We _could_ also add another sentence like:
```
Use `sushi {command} help` to get information about a specific command and its options (e.g., `sushi build help`).

Reply:
You know, I almost reworked this and then talked myself out of it. I like the extra sentence about the help for a command, so I'll add that.

Reply:
I tweaked the sentence a bit in [adc57bf](https://github.com/FSHSchool/site/pull/90/commits/adc57bfe3ccfce273ebc4af6c3502cf246796d46). Let me know what you think.